apps/webClient/src/components/NavButton/index.tsx

```diff
@@ -1,43 +1,65 @@
 import React from "react";
+import clsx from "clsx";
 import { makeStyles } from "@material-ui/styles";
 import Button from "@material-ui/core/Button";
-import Typography from "@material-ui/core/Typography";
 import { Theme } from "@material-ui/core/styles/createMuiTheme";
 
 interface Props {
   route: string;
   text: string;
+  vertical?: boolean;
+  isLeft?: boolean;
 }
 
 const useStyles = makeStyles((theme: Theme) => ({
-  ourWork: {
-    backgroundColor: "white",
+  verticalBtn: {
+    backgroundColor: "transparent",
     borderRadius: 0,
+    border: "solid 3px white",
     width: 60,
     height: 200,
+    padding: "30px 10px",
+    textTransform: "uppercase",
   },
-  aboutUs: {
+  horizontalBtn: {
     backgroundColor: "transparent",
     borderRadius: 0,
-    border: "solid 4px white",
-    width: 60,
-    height: 200,
+    border: "solid 3px white",
+    width: 200,
+    height: 60,
+    textTransform: "uppercase",
+    fontSize: 18,
   },
-  buttonLabel: {
+  verticalLabel: {
     writingMode: "vertical-rl",
     textOrientation: "sideways",
-    fontSize: 20,
-    textTransform: "uppercase",
+    fontSize: 18,
+  },
```

Comment:
it probably makes sense to do a `button` or `baseButton` class, but we can do that later if we want -- this is a fantastic start